### DISCUSSION TITLE
Variable aws-machine-image datasource

### DISCUSSION THREAD

**Discussion by roee-landesman** [renovatebot/renovate#17118] [Request Help]
Hey ya'll,

I love renovatebot and really want to get it setup with Terraform so that it can open PRs to update AMI versions in our EKS module. I understand that the current setup requires a comment to specify some parameters about the AMI itself (region, type, etc.), but since variables cannot be interpolated inside TF comments i'm not sure how to proceed.

What I'm trying to get at is this:

```
# amiFilter=[{"Name":"owner-id","Values":["${var.environment}"]},{"Name":"name","Values":["amazon-eks-node-1.21-*"]}]
# currentImageName=unknown
my_ami1: ami-02ce3d9008cab69cb
```

**Comment by rarkins** [comment]
If you're using a regex manager then you'll need to embed this information inside the renovate config instead.

For further questions please create a public reproduction including the config you're trying
